pkg/controller/user/index.go

```diff
@@ -27,6 +27,9 @@ func (c *Controller) HandleIndex() http.Handler {
 		ctx := r.Context()
 		flash := flash.FromContext(w, r)
 
+		// TODO(crwilcox): this should be moved to a cron run.
+		c.db.UpdateUserStats()
```

Comment:
Do you plan to address the TODO in this PR or a follow up? If follow up, can we wrap this in a query param for now?

Reply:
@sethvargo was split on this. From our daily sync I am going to work in the background cron.